internal/status/status.go

```diff
@@ -0,0 +1,76 @@
+package status
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/filters"
+	"github.com/supabase/cli/internal/utils"
+)
+
+var ctx = context.Background()
+
+func Run(useShortId bool, showKeys bool) error {
+	// Sanity checks.
+	if err := utils.AssertDockerIsRunning(); err != nil {
+		return err
+	}
+	if err := utils.LoadConfig(); err != nil {
+		return err
+	}
+
+	// List containers.
```

Comment:
Let's drop the containers list for now - this might be confusing for new users, and makes the output noisy. Can keep the `Supabase stopped` output though :+1: